rust/lance-io/src/object_store.rs

```diff
@@ -707,14 +696,6 @@ async fn configure_store(url: &str, options: ObjectStoreParams) -> Result<Object
         "s3" | "s3+ddb" => {
             storage_options.with_env_s3();
 
-            // if url.scheme() == "s3+ddb" && options.commit_handler.is_some() {
```

Comment:
I think we should reenable this instead of delete it

Reply:
Sure , will do